### DISCUSSION TITLE
can we connect metamask enabled dapp to walletconnect app ?

### DISCUSSION THREAD

**Discussion by DUR5369** [WalletConnect/walletconnect-monorepo#842] [Developer Support]
I am trying to connect metamask enabled dapp (via moralis)  in my web3 wallet  application that has dev side integration
is it possible
its a custom chain over geth

**Comment by Ztvampire** [comment]
y

**Reply by DUR5369** [reply]
The problem is 
the dapp has installed moralis and authenticated wallet with wallet connect but still it show balance zero why so 
please help.

**Comment by lkh0482** [comment]
y
